### PR TITLE
Feature/custom query

### PR DESCRIPTION
**Pull request by MattPoblete**
This pull request solves `https://github.com/orgs/soroswap/projects/4/views/1?pane=issue&itemId=50793996`

- Add getCustomQuery & subscribeToMultipleLedgerEntries methods on mercury.ts

### CODE REVIEW

File: src/Mercury.ts [hunk 1]
```diff
@@ -193,15 +195,44 @@ export class Mercury {
    * Create a new subscription to a ledger entry. This is especially useful in scenarios where events alone don't give you enough context.
    * @param args Arguments for the subscription:
    *   - contractId: ID of the contract.
-   *   - keyXdr: Entry key as base64 xdr
+   *   - keyXdr: Entry key as base64 xdr.
+   *   - durability: Durability of the entry.
+   *   - keyXdr: Entry key as base64 xdr.
```
Comment (chopan123) [inline src/Mercury.ts]: duplicated docs

File: src/Mercury.ts [hunk 2]
```diff
@@ -193,15 +195,44 @@ export class Mercury {
...
+   *   - durability: Durability of the entry.
    *   - maxSingleSize (optional): How much will one event cost at most (default: 2000)
    * @returns Subscription result.
    */
   public async subscribeToLedgerEntries(args: SubscribeToLedgerEntriesArgs) {
     const body = this._createRequestBody(args, {
       maxSingleSize: this._defaultMaxSingleSize,
```
Comment (chopan123) [inline src/Mercury.ts:205]: you may need something like:
```typescript
const body = this._createRequestBody(args, {
      durability: args.durability,
      key_xdr: args.keyXdr,
      maxSingleSize: this._defaultMaxSingleSize,
      }
  ```

**Comment by MattPoblete** [comment]
- Fix subscribeToLedgerEntries & subscribeToMultipleLedgerEntries demos by correcting keyXdr value on function call args.
- Correct misspelled field names in types.

File: src/Mercury.ts (hunk 1 above)
Comment (MattPoblete) [reply src/Mercury.ts]: Removed duplicated docs!
image

File: src/Mercury.ts (hunk 2 above)
Comment (MattPoblete) [reply src/Mercury.ts:205]: image
Updated request body structure.

**Comment by MattPoblete** [comment]
Fix valueXdr field on single subscribeToLedgerEntries args
image

**Review by chopan123 — APPROVED** [review]

**Pull request merged**